types/office-js-preview/index.d.ts

```diff
@@ -91152,6 +91690,16 @@ declare namespace Word {
          * [Api set: WordApi 1.5]
          */
         getFootnoteBody(): Word.Body;
+        /**
+         * Gets the paragraph by its unique id. Throws an `ItemNotFound` error if the collection is empty.
+         *
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param id Required. Unique Paragraph Id.
```

Comment:
If this is a GUID, specify the required formatting. E.g., are there hypens in it? Are there {} around it?

Reply:
Updated param description